### PR TITLE
[Merged by Bors] - feat: `a • t ⊆ s • t`

### PR DESCRIPTION
**Pull request by YaelDillies**
Match https://github.com/leanprover-community/mathlib/pull/18697

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Finset/Pointwise.lean [hunk 1]
```diff
@@ -1567,10 +1605,10 @@ theorem coe_smul_finset (a : α) (s : Finset β) : ↑(a • s)  = a • (↑s :
 #align finset.coe_vadd_finset Finset.coe_vadd_finset
 
 @[to_additive]
-theorem smul_finset_mem_smul_finset : b ∈ s → a • b ∈ a • s :=
+theorem smul_mem_smul_finset : b ∈ s → a • b ∈ a • s :=
   mem_image_of_mem _
-#align finset.smul_finset_mem_smul_finset Finset.smul_finset_mem_smul_finset
-#align finset.vadd_finset_mem_vadd_finset Finset.vadd_finset_mem_vadd_finset
+#align finset.smul_finset_mem_smul_finset Finset.smul_mem_smul_finset
+#align finset.vadd_finset_mem_vadd_finset Finset.vadd_mem_vadd_finset
```
Comment (eric-wieser) [inline Mathlib/Data/Finset/Pointwise.lean:1611]: Shouldn't the Lean3 names be changing here too?

**Review by eric-wieser — APPROVED** [review]
bors d+

One `align` statement hasn't been forward-ported correctly.

File: Mathlib/Data/Finset/Pointwise.lean (hunk 1 above)
Comment (Parcly-Taxel) [inline Mathlib/Data/Finset/Pointwise.lean:1611]:
```suggestion
#align finset.smul_mem_smul_finset Finset.smul_mem_smul_finset
#align finset.vadd_mem_vadd_finset Finset.vadd_mem_vadd_finset
```

**Review by eric-wieser — APPROVED** [review]
bors merge

**Pull request closed without merging**